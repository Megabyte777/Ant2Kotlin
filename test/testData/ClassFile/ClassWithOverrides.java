package testData.ClassFile;

public class ClassWithOverrides {
    public ClassWithOverrides() {}
    public ClassWithOverrides(int a) {}
    public ClassWithOverrides(String a) {}

    public void method1() {}

    public int method1(int a) {
        return a;
    }

    public String method1(String s) {
        return s;
    }
}
