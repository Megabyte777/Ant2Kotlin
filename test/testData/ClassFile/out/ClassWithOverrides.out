method1()V
<init>(Ljava/lang/String;)V
<init>(I)V
method1(I)I
method1(Ljava/lang/String;)Ljava/lang/String;
<init>()V
